scripts/templates/values.yaml

```diff
@@ -15,17 +15,67 @@ iq:
   licenseSecret: ""
   # configYaml is the full text of the config.yml file that will be passed to IQ Server
   configYaml:
+    sonatypeWork: /sonatype-work
+    createSampleData: true
+    # add this line with this file path and the `licenseSecret` above to autoconfigure licensing
+    # licenseFile: /etc/nexus-iq-license/license_lic
     server:
       applicationConnectors:
         - type: http
           port: 8070
       adminConnectors:
         - type: http
           port: 8071
-    createSampleData: true
-    sonatypeWork: /sonatype-work
-    # add this line with this file path and the `licenseSecret` above to autoconfigure licensing
-    # licenseFile: /etc/nexus-iq-license/license_lic
+      requestLog:
+        appenders:
+          #All appenders set to console
+          - type: file
+            currentLogFilename: /var/log/nexus-iq-server/request.log
+            # Do not display log statements below this threshold to stdout.
+            # threshold: INFO
+            logFormat: "%clientHost %l %user [%date] \"%requestURL\" %statusCode %bytesSent %elapsedTime \"%header{User-Agent}\""
+            archivedLogFilenamePattern: /var/log/nexus-iq-server/request-%d.log.gz
+            archivedFileCount: 50
+    logging:
+      # The default level of all loggers. Can be OFF, ERROR, WARN, INFO, DEBUG, TRACE, or ALL.
+      level: DEBUG
+      # Logger-specific settings.
+      loggers:
+        "com.sonatype.insight.scan": INFO
+        "eu.medsea.mimeutil.MimeUtil2": INFO
+        "org.apache.http": INFO
+        "org.apache.http.wire": ERROR
+        "org.eclipse.birt.report.engine.layout.pdf.font.FontConfigReader": WARN
+        "org.eclipse.jetty": INFO
+        "org.apache.shiro.web.filter.authc.BasicHttpAuthenticationFilter": INFO # WARNING: This reveals credentials at DEBUG level
+        "com.sonatype.insight.audit":
```

Comment:
One forward looking thought. These are from the IQ config.yml which is good, but eventually they will be stale. Any thoughts on how we grab from the main config.yml

Reply:
noted: https://issues.sonatype.org/browse/INT-5938